### DISCUSSION TITLE
Multiple Media Servers

### DISCUSSION THREAD

**Discussion by JaidenW** [seerr-team/seerr#798] [Ideas]
The one feature that this lacks that would get me to move off of Ombi would be support for multiple media servers. On my setup, i run Emby, Plex & Jellyfin side by side so that the user can chose how they'd like to stream & since this is a fork of overseer, it just works with organizrr too, so it's very close, and i'd very much like to get off of Ombi, because it's not been cooperating lately.

If it was possible to have this support all the different apps at once like ombi does, even showing buttons for each of the 3 platforms on an item, so you can just use it as an all in one UI for requesting and launching content
![I9Mef0C 1](https://github.com/Fallenbagel/jellyseerr/assets/22404396/9b0da3cf-6a27-4143-8c79-90cb0a64ef63)

**Comment by fallenbagel** [comment]
You can follow the issue #100
